.github/workflows/push.yml

```diff
@@ -1,59 +1,43 @@
-name: Test
+name: Tests
 on:
   push:
     branches:
-      - 1.x #CraftCMS v3 | PHP 7.2
       - 2.x #CraftCMS v4 | PHP 8.0.2
   pull_request:
     branches:
       - "*"
 jobs:
-  tests-php-72:
+  tests-php-80:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Set the value
         id: step_one
         run: |
-          echo "PHP_VERSION=7.2" >> $GITHUB_ENV
-      - name: Build containers
-        run: make up
-      - name: Composer install
-        run: make composer-install
-      - name: Tests
-        run: make test
-
-  tests-php-73:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set the value
-        id: step_one
-        run: |
-          echo "PHP_VERSION=7.3" >> $GITHUB_ENV
+          echo "PHP_VERSION=8.0" >> $GITHUB_ENV
       - name: Build containers
         run: make up
       - name: Composer install
         run: make composer-install
       - name: Tests
         run: make test
 
-  tests-php-74:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set the value
-        id: step_one
-        run: |
-          echo "PHP_VERSION=7.4" >> $GITHUB_ENV
-      - name: Build containers
-        run: make up
-      - name: Composer install
-        run: make composer-install
-      - name: Tests
-        run: make test
+#  tests-php-81:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Set the value
+#        id: step_one
+#        run: |
+#          echo "PHP_VERSION=8.1" >> $GITHUB_ENV
+#      - name: Build containers
+#        run: make up
+#      - name: Composer install
+#        run: make composer-install
+#      - name: Tests
+#        run: make test
```

Comment:
just curious, why is this one commented out?

Reply:
it can be safety removed, since it is leftover from craft v3 branch, in craft v4 we support only 2 php versions 8.0 and 8.1 😉 

I will remove this, thanks!

Reply:
done